pr-progress/pr-progress.md

```diff
@@ -0,0 +1,42 @@
+# PR Progress Updates
```

Comment:
we should remove that file before merging, @hariombalhara can you make sure that the found bugs are not getting lost and issues are created for them?

Reply:
Done and filed bugs. One of the fixes got merged today itself.